user.js

```diff
@@ -675,49 +675,55 @@ module.exports = function user (options) {
 
       login.save$(function (err, login) {
         if (err) return done(err)
-        seneca.act({role: role, cmd: 'set_user_lock', id: user.id}, function (err, out) {
+        cmd_increment_lock(seneca, user.id, true, function (err, out) {
           if (err) return done(err)
+
           seneca.log.debug('login/ok', why, user, login)
           done(null, {ok: true, user: user, login: login, why: why})
```

Comment:
added fail count in the answer just for info purposes, thinking that it is not security critical info. I could remove it although I don't see the issue with it